Sources/TripKit/helpers/TKUserProfileHelper.swift

```diff
@@ -132,5 +133,73 @@ public class TKUserProfileHelper: NSObject {
     }
   }
   
+  // MARK: - Mode by mode, Mode picker
+  
+  @objc public class func isSharedVehicleModeEnabled(identifier: ModeIdentifier) -> Bool {
+    return disabledSharedVehicleModes.contains(identifier)
+  }
+  
```

Comment:
Would be best to remove this function as just have the public interface based on `TKModeInfo` for simplicity. 

Also, if you added a regular mode identifier to `disabledSharedVehicleModes`, then calling `isSharedVehicleModeEnabled(mode:)` can crash if there's no ":" in the identifier and therefore no `components[1]`.

```suggestion
```

Reply:
oh okay, noted on this one.